ansible/playbook.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Deploy https://girder.dandiarchive.org/
+  hosts: all
+  vars:
+    ansible_python_interpreter: auto
+
+  roles:
+    - role: girder.mongodb
+      vars:
+        mongodb_data_path: /var/lib/mongodb
+    - role: girder.girder
+    - role: girder.nginx
+      vars:
+        nginx_hostname: "girder.dandiarchive.org"
+        nginx_registration_email: "michael.grauer@kitware.com"
+
+  tasks:
+    - name: Install awscli
+      apt:
+        name: awscli
+        update_cache: true
+      become: true
+
+    - name: Create .aws directory
+      file:
+        path: "{{ ansible_env.HOME }}/.aws"
+        state: directory
+
+    - name: Copy aws config file
+      copy:
+        src: files/config
+        dest: "{{ ansible_env.HOME }}/.aws/config"
+
+    - name: Install Girder plugins
+      pip:
+        name:
+          - girder-oauth
+          - girder-sentry
+          - girder-hashsum-download
```

Comment:
should we add some analytics (google or just downloads) right away?

Reply:
Good question.

Google analytics (the Girder plugin) will track interactions with the Girder web client, which we won't expose to end users, so there isn't a need for that plugin to be enabled.

The download statistics Girder plugin will track the number of times a file or item has been downloaded, but no more detailed statistics than that. We could enable this one very easily, but I'd like us to have clear requirements and expectations around any download stats tracking.

Reply:
would frontend have its own log (for future analysis, alike apache logs) of interactions with it?

Reply:
the frontend currently doesn't have that. we would need to clarify requirements there if that is desired. it definitely won't happen before SfN.

Reply:
I will add download statistics plugin, see #11.